### DISCUSSION TITLE
How do I hide certain table columns when printing?

### DISCUSSION THREAD

**Discussion by nexxai** [filamentphp/filament#14449] [Help]
### Package

Panel builder

### Package Version

v3.2.110

### How can we help you?

So I know about the "print:hidden" class for Tailwind but what I'm not clear on is how I can actually apply it to certain columns. 

Using the instructions [here](https://filamentphp.com/docs/3.x/tables/columns/getting-started#custom-attributes), I tried adding `->extraAttributes(['class' => 'print:hidden'])` to the columns I want to hide, but they're still showing up so either I'm doing it wrong, or it doesn't work like I'm assuming.

I've also tried `->hiddenFrom('print')` but then I get an `UnhandledMatchError: Unhandled match case 'print'`

EDIT: So I published and updated the `filament-tables/index.blade.php` stub to include `print` as a breakpoint, and now it's actually showing the class where it should, but for some reason, my browser literally isn't respecting the class.  The crazy part is that i use the same `print:hidden` class elsewhere on my site and it works fine.  Is there something about Filament that overwrites print-specific breakpoint classes?

```php
if ($breakpoint = $column->getHiddenFrom()) {
    return match ($breakpoint) {
        'print' => 'print:hidden',             <--- added this on line 115
        'sm' => 'sm:hidden',
        'md' => 'md:hidden',
        'lg' => 'lg:hidden',
        'xl' => 'xl:hidden',
        '2xl' => '2xl:hidden',
    };
}
```

**Comment by leandrocfe** [comment]
I think you need to create a [custom theme](https://filamentphp.com/docs/3.x/panels/themes#creating-a-custom-theme)
_If you want to apply Tailwind CSS classes that are not already used in Blade files, you should update your content configuration in tailwind.config.js to also scan for classes inside your directory: './app/Filament/**/*.php'_
